bridges/AmazonBridge.php

```diff
@@ -72,6 +72,9 @@ public function collectData() {
 
 			// Title
 			$title = $element->find('h2', 0);
+			if ($title == null) {
```

Comment:
Thanks for the PR!

One small adjustment:

```suggestion
			if ($title === null) {
```

or

```suggestion
			if (is_null($title)) {
```